changes/145.feature

```diff
@@ -0,0 +1 @@
+Add host health check command and SDK.
```

Comment:
Let's mention the command, `backend.ai manager health-check`.

Reply:
Appended command to the news fragment.